kube-scheduler-healthcheck

```diff
@@ -10,23 +12,36 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: ${TEST_POD_NAME}
+  labels:
+    app: scheduler-healthcheck-test-pod
 spec:
   containers:
   - name: pause
     image: ${TEST_POD_IMAGE}
 endOfPodDef
 
-alias kubectl="timeout -t 30 kubectl --namespace \$TEST_POD_NAMESPACE"
+alias kubectl="timeout -t 30 kubectl --namespace \${TEST_POD_NAMESPACE}"
+function create-test-pod {
+  kubectl create -f /tmp/pod.yaml
+}
+function get-test-pod {
+  kubectl get pod ${TEST_POD_NAME} "$@"
+}
+function delete-test-pod {
+  kubectl delete pod --selector='app=scheduler-healthcheck-test-pod' --grace-period=0 "$@"
```

Comment:
Won't this now delete the pods from other instances of the healthcheck running at the same time?

This might then make those healthchecks break, and the instances be killed without reason

Reply:
👍